### PR TITLE
COMMERCE-817 Make sure old urls will show the new product

### PR DESCRIPTION
**Pull request by alecsloan**
Let me know if you have any questions.

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test

**Comment by ethanbustad** [comment]
Hey @alecsloan, found another instance of that bug here: https://github.com/liferay/com-liferay-commerce/blob/7.1.x-next/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/portlet/AssetCategoryFriendlyURLResolver.java#L97. Can you fix this one, too?

File: commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/portlet/AssetCategoryFriendlyURLResolver.java [hunk 1]
```diff
@@ -140,7 +140,7 @@ public String getActualURL(
 			assetCategory.getDescription(languageId), httpServletRequest);
 
 		List<AssetTag> assetTags = _assetTagLocalService.getTags(
-			AssetCategory.class.getName(), assetCategory.getPrimaryKey());
+			AssetCategory.class.getName(), assetCategory.getClassPK());
```
Comment (alecsloan) [inline commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/portlet/AssetCategoryFriendlyURLResolver.java]: @ethanbustad This should have been classPK aswell according to [javadoc](https://docs.liferay.com/portal/7.0/javadocs/portal-kernel/com/liferay/asset/kernel/service/AssetTagLocalServiceUtil.html#getTags-java.lang.String-long-)

**Comment by alecsloan** [comment]
> Hey @alecsloan, found another instance of that bug here: https://github.com/liferay/com-liferay-commerce/blob/7.1.x-next/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/portlet/AssetCategoryFriendlyURLResolver.java#L97. Can you fix this one, too?

[Found another one](https://github.com/liferay/com-liferay-commerce/pull/373/files#r246945654)

File: commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/portlet/AssetCategoryFriendlyURLResolver.java (hunk 1 above)
Comment (alecsloan) [reply commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/portlet/AssetCategoryFriendlyURLResolver.java]: disregard

**Comment by marco-leo** [comment]
Merged! Thanks

**Pull request merged**